docs/loading/S3_Load.md

```diff
@@ -0,0 +1,272 @@
+# Load data from Amazon S3
+
+StarRocks provides two options for loading data from S3:
+
+1. Asynchronous loading using Broker Load (`WITH BROKER`)
+2. Synchronous loading using the `FILES()` table function
+
+Small datasets are often loaded synchronously using the `FILES()` table function, and large datasets are often loaded asynchronously using Broker Load. The two methods have different advantages and are described below.
+
+## Using Broker Load
+
+An asynchronous Broker Load process handles making the connection to S3, pulling the data, and storing the data in StarRocks.
+
+### Advantages of Broker Load
+
+- Broker Load supports data transformation, UPSERT, and DELETE operations during loading.
+- Broker Load runs in the background and clients don't need to stay connected for the job to continue.
+- Broker Load is preferred for long running jobs, the default timeout is 4 hours.
+- In addition to Parquet and ORC file format, Broker Load supports CSV files.
+
+### Data flow
+
+![Workflow of Broker Load](../assets/broker_load_how-to-work_en.png)
+
+1. The user creates a load job
+2. The frontend (FE) creates a query plan and distributes the plan to the backend nodes (BE)
+3. The backend (BE) nodes pull the data from the source and load the data into StarRocks
+
+### Typical example
```

Comment:
A few thoughts:
1) Below we have only one example showing how to load data from a single file to a single table. Users have complained about that because they want to see more examples, one for each of the load scenarios that StarRocks supports: loading from a single file to a single table, loading from multiple files (that is, all files in a specific path) to a single table, and loading from multiple files to multiple tables (for examples, file A to table A and file B to table B). And that's why in v3.1 docs we have added these examples. Can we include these in the new s3 loading doc?
2) Typically CSV is the most common file format I think? So is it better if we use CSV as an example to describe to perform loads in various loading scenarios? And then we also provide some instructions on how to load Parquet and ORC format files?

Reply:
@alberttwong do you have a feeling about popularity of Parquet vs. CSV, etc? While I like CSV for examples of I am telling the reader to create a file by hand, I don't know if people use them for large datasets.

Reply:
I don't really hear about CSV.  It's almost all parquet.

Reply:
Thanks @alberttwong , similar question: It seems like we refer to all separated value file formats as CSV.  In my experience, CSV is just comma separated, and TSV is tab separated.  Do you get questions about support for TSV?